### ISSUE TITLE
Correct dynamicProperty comment

### ISSUE THREAD

**Issue by timrobertson100** [tdwg/dwc#78]
http://rs.tdwg.org/dwc/terms/#dynamicProperties provides examples which are invalid JSON (they are enclosed in quotes, and have quotes within).  Suggest using a style to emphasise the example, and not using the enclosing quotes.

**Comment by mdoering** [comment]
good idea, we do quote all examples everywhere, but a style makes it so much simpler!
Ah, now we only need to figure out how to parse the (https://github.com/tdwg/dwc/blob/dwc_cleanup/terms/dwc_normative.rdf#L395)[RDF comment] string and turn that into html markup...

**Comment by mdoering** [comment]
I would propose to quote all examples in RDF using triple single quotes, e.g. '''this is quoted'''

**Comment by peterdesmet** [comment]
Can't we use `<code></code>`? To be that seems the best way to show verbatim information. I'd like to render it that way on the website too.

**Comment by tucotuco** [comment]
In the RDF? We should not have extra encoding there. It should appear like
a correct example as text.


This, we just need to remove the double quoted enclosing each distinct
example, correct?  For markup, sure, <code> sounds good to me.

**Comment by mdoering** [comment]
Would it make sense to keep each example as a distinct rdf triple with a "dwcattributes:example" predicate or sth like that?

**Comment by tucotuco** [comment]
The reaons being it would be easier to parse the rdf to make the
documentation? I think it would end u complicating thing elsewhere, such as
the Vocab management Wiki. I would rather keep them in the comments I think.

**Comment by peterdesmet** [comment]
I don't mind how they are stored in the RDF, but I'll try to display them better on the web page.

**Comment by mdoering** [comment]
... which requires parsing them out. ''', <em>, <code> or individual triples are on the table. I think I prefer to avoid having html tags inside the rdf, but I have no strong opinion

**Comment by tucotuco** [comment]
Amended proposal to follow Ramona's advice to include a dwcattributes annotation for examples. Reset milestone to 5 Mar 2015.

**Comment by tucotuco** [comment]
Awaiting outcome of Issue #26 for the question of parsing examples. If an example attribute is recommended, with one example per instance of the attribute, that solves the issue of creating markup. It will also solve the problem of rendering the JSON example which initiated this particular issue. Thus, changing the milestone to correspond with the extended commentary period for Issue #26.

**Comment by peterdesmet** [comment]
Update: examples (in recommendations) have been moved out of the standard (#26) to `build/config/terms.csv` and the normative document is a CSV anyway, so we have more options on how to encode examples. We opted for `backticks` as originally suggested in https://github.com/tdwg/dwc/issues/78#issuecomment-70544951, which are rendered as `<code>`. Implemented in b998b9a77bb7e67931f00546d1af655b42ddd60b.

The examples for `dynamicProperties` (the original topic of this issue) look better now:

<img width="813" alt="dynamicproperties" src="https://user-images.githubusercontent.com/600993/31312107-0273fcba-ab89-11e7-95df-d016f0a82422.png">

Guide is not public yet, but I'm closing this issue.
